classes/class-wc-connect-shipping-label.php

```diff
@@ -56,7 +56,7 @@ protected function get_packaging_data( WC_Order $order ) {
 			$shipping_methods = $order->get_shipping_methods();
 			$shipping_method = reset( $shipping_methods );
 			if ( ! $shipping_method || ! isset( $shipping_method[ 'wc_connect_packages' ] ) ) {
-				return $this->get_items_as_individual_packages( $order );
+				return array( FALSE, $this->get_items_as_individual_packages( $order ) );
```

Comment:
I had to look at how this method was used to understand what `TRUE` and `FALSE` where.
How about returning `array( 'is_packed' => FALSE, 'packages' => .... )` ?